src/components/Navbar/index.spec.tsx

```diff
@@ -23,18 +23,23 @@ import Navbar from '.';
 describe(__filename, () => {
   type RenderParams = {
     _requestLogOut?: typeof requestLogOut;
+    reviewersHost?: string;
     store?: Store;
   };
 
   const render = ({
     _requestLogOut = jest.fn(),
+    reviewersHost,
     store = configureStore(),
   }: RenderParams = {}) => {
     // TODO: Use shallowUntilTarget()
     // https://github.com/mozilla/addons-code-manager/issues/15
-    const root = shallow(<Navbar _requestLogOut={_requestLogOut} />, {
-      context: { store },
-    }).shallow();
+    const root = shallow(
```

Comment:
It looks like you can switch this to `shallowUntilTarget()` and remove the comment up above.